### ISSUE TITLE
Why did you choose learning rate scheduler in that way?

### ISSUE THREAD

**Issue by jiminbot20** [zhouhaoyi/Informer2020#404]
I just wonder why you made learning rate decrease in hald every 2 epoches like StepLR.

you could fix learning rate or use pytorch's other scheduler. 

Could you explain why? It will be grateful if you explain.

**Comment by zhouhaoyi** [comment]
We just use a simple learning rate decrease, but we have not tests other scheduler.
